Eatery/Controllers/BRB/BRBViewController.swift

```diff
@@ -110,8 +157,12 @@ extension BRBViewController: BRBLoginViewControllerDelegate {
     
 }
 
-extension BRBViewController: BRBConnectionDelegate {
+extension BRBViewController: BRBAccountManagerDelegate {
     
+    func queriedAccount(account: BRBAccount) {
```

Comment:
I tried  pulling your branch and could not get it to compile because this function is incomplete, make sure to check that everything compiles before submitting a PR!

Reply:
I think this may have been caused by a merge conflict. Git will try its best to combine master with your branch, but it may not do a perfect job. Sometimes you'll get merge conflicts, other times it'll do things like this. I usually run through all the changes and try to compile the app before submitting PRs for this reason

Reply:
Yes, I saw that there was a merge conflict in this function, which I believe I fixed